tests/test_apply.py

```diff
@@ -454,6 +454,18 @@ def test_load_default_qubit_device(self):
         assert dev.analytic
         assert dev.short_name == "lightning.qubit"
 
+    def test_no_backprop(self):
+        """Test that lightning.qubit does not support the backprop
+        differentiation method."""
+
+        dev = qml.device("lightning.qubit", wires=2)
+        def circuit():
+            """Simple quantum function."""
+            return qml.expval(qml.PauliZ(0))
+
+        with pytest.raises(qml.QuantumFunctionError):
+            lightning = qml.QNode(circuit, dev, diff_method="backprop")
+
```

Comment:
Maybe also worth having a test that checks that `diff_method="best"` results in `isinstance(qnode.device, LightningQubit)`? I'm a bit paranoid now 😆

Reply:
Good idea, added!